### DISCUSSION TITLE
help me to resolve this error

### DISCUSSION THREAD

**Discussion by vijaypotnuru** [open-sauced-craftwork/browser-extension#29] [General]
![image](https://user-images.githubusercontent.com/110402074/225925940-582cf378-bd75-4164-845b-d30ac1567d4c.png)

![image](https://user-images.githubusercontent.com/110402074/225926190-1fec7594-a786-4a64-81f2-8bd15adc1e4e.png)

![image](https://user-images.githubusercontent.com/110402074/225926340-4caa056d-db8c-48c3-9e74-c2d4be8d6aa0.png)

![image](https://user-images.githubusercontent.com/110402074/225926441-edb49818-1f8a-48bb-9a96-bf9cdf633a94.png)

**Comment by vijaypotnuru** [comment]
i follow this step from but extension not working

https://crxjs.dev/vite-plugin/getting-started/vanilla-js/create-project

https://crxjs.dev/vite-plugin/getting-started/vanilla-js/create-project


please me to find the error

**Comment by nickytonline** [comment]
I don't know what's wrong as I can't see any errors, but if you click on the `Errors` button for your extension, you should have more information to help debug this.

![CleanShot 2023-03-17 at 10 10 11](https://user-images.githubusercontent.com/833231/225928870-de349a62-0195-4ae0-b5b5-075552b08cfb.png)

**Reply by vijaypotnuru** [reply]
how to share my code to you sir 
should i do pull request

**Reply by nickytonline** [reply]
Before we look at any code, can you share the error displayed when you click the `Errors` button?

**Reply by vijaypotnuru** [reply]
![image](https://user-images.githubusercontent.com/110402074/225946084-7d48a42f-1478-46fe-9397-1b135c002e4c.png)

i check the error in service-worker-loader.js that was url error the forward slash is this then i add the forward slash into url but when i run the npm run dev  in terminal automatically forward slash removed

see this :
![image](https://user-images.githubusercontent.com/110402074/225948661-34b9e9e5-259b-4d85-9f18-3876a566d1df.png)

**Comment by akash-singh8** [comment]
@vijaypotnuru It's a simple error and to solve this you just need to run the local server using `npm run dev` and you'll be good to go. This happens because the service worker needs to import some data from localhost:3000 so that's why we need to run the local server parallelly in order to work with the browser extension.

**Reply by vijaypotnuru** [reply]
But bro app.jsx not running

**Reply by akash-singh8** [reply]
By not running, what did you mean? because I don't think it has anything to do with this error

**Reply by vijaypotnuru** [reply]
If you don't mind can you create new folder in your desktop and follow this steps from this 

https://dev.to/jacksteamdev/create-a-vite-react-chrome-extension-in-90-seconds-3df7

Your can find the error

Now what is the problem is, the content of app.jsx file not showing in extension please

**Reply by takanome-dev** [reply]
I was having this error, but after closing and re-opening the extension it works again. Or maybe refresh/restart the extension in `chrome://extensions/`

**Reply by Inzamamdev** [reply]
Hey @vijaypotnuru what is not running in the App.jsx can you elaborate your error

**Reply by vijaypotnuru** [reply]
the solved thank for all

**Reply by akash-singh8** [reply]
@vijaypotnuru If it's solved you can mark this as the answer, so others can get help from it.
